src/main/java/org/kiwiproject/test/mongo/MongoServerTests.java

```diff
@@ -15,10 +16,23 @@
     /**
      * Start a new in-memory {@link MongoServer}.
      *
+     * Defaults server version to Mongo 3.6
+     *
      * @return the started MongoServer instance
      */
     public static MongoServer startInMemoryMongoServer() {
```

Comment:
Maybe we should remove this method and force use of the new one with the `ServerVersion` argument? I guess that would -- annoyingly -- technically cause a breaking API change and force a major version update. Should we deprecate it for removal, or not worry about it?

Reply:
I left it in case for some reason someone wanted to start their own in memory mongo db without the extension.